app/src/main/java/org/mozilla/fenix/utils/ClipboardHandler.kt

```diff
@@ -63,6 +60,13 @@ class ClipboardHandler(val context: Context) {
         }
     }
 
+    /**
+     * We cannot rely on `isPrimaryClipEmpty()` since it triggers a clipboard access notification.
+     */
+    fun containsText(): Boolean {
+        return clipboard.isPrimaryClipHtmlText() || clipboard.isPrimaryClipPlainText() || clipboard.isPrimaryClipUrlText()
```

Comment:
In the `run-detekt` task below, you will see in the logs that we have a lint error for `MaxLineLength`. Log available at https://github.com/mozilla-mobile/fenix/actions/runs/3272855315/jobs/5827687650#step:4:171

You can also learn more about our styles at https://github.com/mozilla-mobile/firefox-android/tree/main/android-components#style.